app/controllers/home_controller.rb

```diff
@@ -4,4 +4,13 @@ class HomeController < ApplicationController
   def index
     @users = User.all
   end
+
+  def destroy
+    @user = User.find(params[:id])
```

Comment:
Since you're not authorizing this action, it's easy to send a request to that particular node and delete anyone, including yourself. You just need to be logged in, regular user to do that.